awx_collection/test/awx/test_completeness.py

```diff
@@ -30,7 +30,7 @@
 
 # Global module parameters we can ignore
 ignore_parameters = [
-    'state', 'new_name', 'update_secrets'
+    'state', 'new_name', 'update_secrets', 'copy_from'
```

Comment:
I'd like to see if we can actually inspect endpoints for related/copy URLs and then report if the module should support copy or not. But that can be a different PR.

Reply:
agreed, that would be a good feature, will think about it :)